src/classes/User.ts

```diff
@@ -21,63 +35,70 @@ export default class User {
         this.#appID = config.appID ? config.appID : '';
         this.#apiKey = config.apiKey ? config.apiKey : '';
         this.id = '';
+        this.#authorizationHeader = {
+            'Passage-Version': passageNodeConfig.version,
+        };
+
+        this.#configuration = new Configuration({
+            accessToken: this.#apiKey,
+            apiKey: this.#apiKey,
```

Comment:
I think `accessToken` will add the proper header, so the `apiKey` setting wouldn't be needed:
```suggestion
```

Same for the configurations in the other class.